config/install/field.field.node.localgov_page.localgov_page_banner.yml

```diff
@@ -21,6 +22,7 @@ settings:
   handler: 'default:paragraph'
   handler_settings:
     target_bundles:
+      localgov_banner_primary: localgov_banner_primary
       localgov_banner_secondary: localgov_banner_secondary
     negate: 0
     target_bundles_drag_drop:
```

Comment:
I don't think you need all the disabled target_bundles_drag_drop items; they are referencing paragraph types that may not exist.

I suggest removing all the items with enabled: false, shortening this to
```yaml
    target_bundles_drag_drop:
      localgov_banner_primary:
        weight: 0
        enabled: true
      localgov_banner_secondary:
        weight: 1
        enabled: true
```